source/GridListImageItem.js

```diff
@@ -33,6 +33,20 @@
 
 		/**
 		* @private
+		* @lends moon.Button.prototype
+		*/
+		published: {
+			/**
+			* @type {String}
+			* @default ''
```

Comment:
The default value in the documentation should be updated to reflect the current value.